src/applications/disability-benefits/686c-674/analytics/helpers.js

```diff
@@ -0,0 +1,23 @@
+import recordEvent from 'platform/monitoring/record-event';
+
+export const fireAnalytics = submission => {
+  recordEvent({
```

Comment:
This could be made a little more readable a `const options = submission['view:selectable686Options'];` was defined here.